src/main/java/net/mcreator/ui/modgui/BlockGUI.java

```diff
@@ -1249,15 +1249,19 @@ private void refreshBonemealProperties() {
 
 	private void updateTextureOptions() {
 		texture.setVisible(false);
+		texture.setFlipOnX(false);
 		textureTop.setVisible(false);
+		textureTop.setFlipOnX(false);
 		textureLeft.setVisible(false);
 		textureFront.setVisible(false);
 		textureRight.setVisible(false);
 		textureBack.setVisible(false);
 
 		if (normal.equals(renderType.getSelectedItem())) {

```

Comment:
Grass model should probably also flip the texture

Reply:
Thinking about this, I got a question: do textures only flip on X axis or rotate by 180 degrees? Asking because I'm not dealing with that very often and could be wrong here.

Reply:
Klemen, I just checked a custom block in the test environment and textures are not only flipping on X axis, but on Y too, which means we were still not converting textures properly. I updated the method to flip the selected image on both axes, but how should it be named after that? `setFlipUV(boolean)`? `setRotate180(boolean)`?

Reply:
If flip UV is the correct term, that this should be ok

Reply:
You still kept grass without flipping, is this intentional?

Reply:
**Normal** model is the only one that requires separate texture for each of 6 sides, others use the same texture for each horizontal side, so if you check your block in the game, you can look at its texture named **Back** in the application and top and bottom textures will be rotated properly for you. Only **Normal** model has a front side, so it's important to adapt its textures to look in the right direction, for others any horizontal side could be the front one.

Reply:
We apply flipUV to bottom and top textures. Grass model also has separate top and bottom textures, doesn't it?

Reply:
But side textures are the same.

Reply:
>If flip UV is the correct term, that this should be ok

I tried using `AffineTransform` so that the method deserves the name `setRotate180(boolean)`, but I messed it up and it did not paint the image the way I wanted when "flipped".

Reply:
I see what you mean now, ok